File: backend/CoffeeAPI/Models/Drink.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace CoffeeAPI.Models
{
    public class Drink
    {
        public Guid DrinkId { get; set; }

        [Required]
        public string drinkName { get; set; }
        public bool Available { get; set; }
        public string ImageUrl { get; set; }
        public bool Additions { get; set; }

        public ICollection<OrderLine> OrderLines { get; set; }
    }
}

File: frontend/src/app/core/classes/drink.ts
export class Drink {
    DrinkId: string;
    drinkName: string;
    Available: boolean;
    ImageUrl: string;
    Additions: boolean;

    constructor(DrinkId: string, drinkName: string, Available: boolean = true, ImageUrl: string = "/assets/Images/Latte Macchiato.jpg", Additions: boolean = true) {
        this.DrinkId = DrinkId;
        this.drinkName = drinkName;
        this.Available = Available;
        if (ImageUrl != null && ImageUrl != "") {
            this.ImageUrl = ImageUrl;
        } else {
            this.ImageUrl = "/assets/Images/Latte Macchiato.jpg";
        }
        this.Additions = Additions;
    }
}
